fault-monitor/operational-status-monitor.cpp

```diff
@@ -110,7 +162,7 @@ void Monitor::updateAssertedProperty(
             // Call "Group Asserted --> true" if the value of Functional is
             // false Call "Group Asserted --> false" if the value of Functional
             // is true
-            PropertyValue assertedValue{!value};
```

Comment:
why do you need to change this ?

Reply:
"functionalValue" is the class member here. so is the change here.